scripts/init-contracts.sh

```
@@ -0,0 +1,110 @@
+error() { printf '%s[init]%s %s\n'  "$C_RED"    "$C_RESET" "$*" >&2; }
+ok()    { printf '%s[init]%s %s\n'  "$C_GREEN"  "$C_RESET" "$*"; }
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+REPO_ROOT="$(cd "$SCRIPT_DIR/.." && pwd)"
+cd "$REPO_ROOT"
+
+load_env_file() {
+  local file="$1"
+  [[ -f "$file" ]] || return 0
+  set -a
+  source <(grep -E '^[A-Za-z_][A-Za-z0-9_]*=' "$file" || true)
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)